tools/filters/sorter.xml

```diff
@@ -5,40 +5,40 @@
         <requirement type="package" version="2.14">grep</requirement>
         <requirement type="package" version="4.4">sed</requirement>
     </requirements>
-    <command detect_errors="aggressive">
-        python '$__tool_directory__/sorter.py'
+    <command detect_errors="exit_code">
```

Comment:
Just to clarify, I think this is not what fixes the tool (in fact [aggressive](https://docs.galaxyproject.org/en/latest/dev/schema.html#error-detection) also fails the job if the exit code is not zero), but the above changes from `subprocess.call()` to `subprocess.check_call()` do.